lib/modules/dosomething/dosomething_campaign/dosomething_campaign.features.field_instance.inc

```diff
@@ -3309,6 +3362,12 @@ The title determines the URL of the page. This means that if you are changing th
 <br/>
 <strong> Limit 25 characters. </strong>');
   t('Official Rules');
+  t('Please use this section to outline some Helpful Tips for the member. Be explicit and stress the simplicity of the process. <br/>
+Here are some commonly used Helpful Tips -- copy and paste and edit ones that are relevant to your campaign into the text field: <br/>
+* Upload your photos as you go. As soon as you have one or more <em> noun verb </em>, take a photo and upload it! <br/>
+* Upload one photo at a time.

```

Comment:
Are we really still worried about this right now? AFAIK our only concern has been keeping global functionality working if something drastic breaks